assets/js/blocks/cart-checkout/cart/cart-line-items-table/cart-line-item-row.tsx

```diff
@@ -235,102 +235,104 @@ const CartLineItemRow = forwardRef< HTMLTableRowElement, CartLineItemRowProps >(
 					) }
 				</td>
 				<td className="wc-block-cart-item__product">
-					<ProductName
-						disabled={
-							isPendingDelete || isProductHiddenFromCatalog
-						}
-						name={ name }
-						permalink={ permalink }
-					/>
-					{ showBackorderBadge ? (
-						<ProductBackorderBadge />
-					) : (
-						!! lowStockRemaining && (
-							<ProductLowStockBadge
-								lowStockRemaining={ lowStockRemaining }
+					<div className={ classnames( 'wc-block-cart-item__wrap' ) }>
```

Comment:
I think we should remove `classnames` here _or_ add a filter to supply a custom wrapper class name. (Don't think there would be a good case for doing this so I'm leaning toward just dropping it)